src/js/game/LevelMVC/Position.js

```diff
@@ -0,0 +1,32 @@
+const FacingDirection = require("./FacingDirection");
+
+const directions = [
+  FacingDirection.North,
+  FacingDirection.East,
+  FacingDirection.South,
+  FacingDirection.West
+];
+
+module.exports = class Position {
+  static add(left, right) {
+    return [left[0] + right[0], left[1] + right[1]];
+  }
+
+  static equals(left, right) {
+    return left[0] === right[0] && left[1] === right[1];
+  }
+
+  static isAdjacent(left, right) {
+    return directions
+        .map(FacingDirection.directionToOffset)
+        .some(offset => Position.equals(Position.add(left, offset), right));
```

Comment:
Tiny style nit: I've been doing line continuations at 2 space indent in this repo, even though we're not consistent about it in the main `code-dot-org` repo.